Sources/Pulse/LoggerMessageStore.swift

```diff
@@ -128,4 +116,5 @@ public final class LoggerMessage: NSManagedObject {
     @NSManaged public var category: String
     @NSManaged public var session: String
     @NSManaged public var text: String
+    #warning("Question: Store metadata as well? https://github.com/apple/swift-log#logging-metadata")
```

Comment:
We should, and the UI should provide you with an option to filter based on metadata. I'm going to add this to my backlog.

Reply:
I'm not sure what the best way to store it would be since I would prefer it to be searchable. So it could maybe be done in a separate PR.

Reply:
This is an example where NoSQL would've worked better...

Reply:
Sounds reasonable. I'd be in favor of moving this into its own Issue/PR.

Reply:
Created an issue https://github.com/kean/Pulse/issues/3